pkg/kubectl/bind/plugin/bind.go

```diff
@@ -303,6 +314,10 @@ func (b *BindOptions) Run(ctx context.Context, urlCh chan<- string) error {
 	return nil
 }
 
+func (opts *BindOptions) Cleanup() {
+	os.Remove(opts.outFile.Name())
```

Comment:
this is strange to be attached to options. Options are supposed to be just a data structure of flag values.